.github/workflows/CI.yaml

```diff
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+      - release/*
+  pull_request:
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        variant: ["''", "core-3x"]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+
+      - name: Configure git
+        run: |
+          git config --local user.email imodeljs-admin@users.noreply.github.com
+          git config --local user.name imodeljs-admin
+
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+
+      - name: Rush install
+        run: node common/scripts/install-run-rush.js install --variant ${{ matrix.variant }}
+
+      - name: Rush build
```

Comment:
Shouldn't `lint` and `prettier` steps also be included in this workflow?

Reply:
Eventually - yes. I tried to keep this minimal and related to unit-tests only.